### PR TITLE
Unacceptable deadlines

### PR DESCRIPTION
**Pull request by burgess01**
### Type of Change

- [ ] Feature
- [ ] Bug fix
- [ x] Documentation

## Contributors

- @burgess01

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by tuduun — APPROVED** [review]
This looks good.

**Review by laurennevill — DISMISSED** [review]
This looks good!

File: conduct.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should be mindful of assignment due dates and work on them accordingly.
+- Students should not ask for extensions without good reason.
+- Students should not contact the professor within 24 hours of a deadline for an extension.
```
Comment (solisa986) [inline conduct.md:51]: Maybe you could add a sentence explaining that deadline extensions should be discussed with the professor privately prior to the deadline.

**Review by solisa986 — CHANGES_REQUESTED** [review]

File: conduct.md (hunk 1 above)
Comment (burgess01) [reply conduct.md:51]: Added this to my branch.

**Review by MAbraham24 — APPROVED** [review]
This is a good change!

**Review by solisa986 — APPROVED** [review]
looks like a good format for the unacceptable deadlines. great work!

**Review by laurennevill — APPROVED** [review]

**Pull request merged**